### DISCUSSION TITLE
Specify GPU index for EGL rendering

### DISCUSSION THREAD

**Discussion by jimmyyhwu** [bulletphysics/bullet3#3445] [General]
On multi-gpu machines, EGL rendering always creates a process on GPU 0. Is there a way to specify (something akin to `CUDA_VISIBLE_DEVICES`) which GPU to use?

**Comment by fxia22** [comment]
You can use `EGL_PLATFORM_DEVICE_EXT`, here is an example:

https://github.com/StanfordVL/iGibson/blob/master/gibson2/core/render/cpp/Mesh_renderer.cpp#L153

https://github.com/StanfordVL/iGibson/blob/master/gibson2/core/render/cpp/Mesh_renderer.cpp#L171-L172
